hugo/content/4-updates/introduction.md

```diff
@@ -0,0 +1,28 @@
++++
+next = "/4-queries/operators"
+prev = "/4-queries"
+title = "Introduction"
+toc = true
+weight = 1
+
++++
+
+Mangrove's query builder also provides the ability to construct **query** expressions.
```

Comment:
s/query/update
(I think this happens a few other times, I'll stop commenting for now though)